converter.go

```diff
@@ -1022,6 +1003,153 @@ func arrowToValue(ctx context.Context, rowIdx int, srcColumnMeta execResponseRow
 	return nil, fmt.Errorf("unsupported data type")
 }
 
+func handleNativeArrowMap(ctx context.Context, rowIdx int, keyMetadata, valueMetadata fieldMetadata, srcValue arrow.Array, loc *time.Location, higherPrecision bool, params map[string]*string) (snowflakeValue, error) {
```

Comment:
`build` instead of `handle`?

Reply:
Ok, done. Needed to change the whole name though.

Reply:
Sure, thanks.